qa/integration/services/elasticsearch_setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
+current_dir="$(dirname "$0")"
+source "${current_dir}/helpers.sh"
+
+if [ -n "${ES_VERSION+1}" ]; then
+  echo "Elasticsearch version is $ES_VERSION"
+  version=$ES_VERSION
+else
+   version=5.0.0-alpha5
+fi
```

Comment:
maybe we could use something like this https://gist.github.com/pkuczynski/8665367 to extract the version from the `versions.yml`?